dojo/fixtures/test_type.json

```diff
@@ -571,6 +571,13 @@
 		"model": "dojo.test_type",
 		"pk": 173
 	},
+        {
+                "fields": {
+                        "name": "Github Vulnerability Scan"
+                },
+                "model": "dojo.test_type",
+                "pk": 178
+        },
```

Comment:
The pk here matches that of BugCrowd on line 609

Reply:
fixed